lib/wicked/wlan.pm

```diff
@@ -261,6 +262,45 @@ sub hostapd_can_wep {
     return $s !~ m/unknown configuration item 'wep_key0'/i;
 }
 
+sub hostapd_can_key_mgmt {
+    my ($self, $key_mgmt) = @_;
+    $self->write_cfg('/tmp/check_key_mgmt.conf', 'wpa_key_mgmt=' . $key_mgmt);
+    my $s = script_output('hostapd /tmp/check_key_mgmt.conf', proceed_on_failure => 1);
+    return $s !~ m/invalid key_mgmt/i;
+}
+
+sub wpa_supplicant_can_key_mgmt {
```

Comment:
```suggestion
sub is_wpa_supplicant_key_mgmt_invalid {
```

Reply:
changed it to `is_wpa_supplicant_supporting_key_mgmt()`